sdk/tests/conformance/context/00_test_list.txt

```diff
@@ -12,6 +12,7 @@ context-attributes-alpha-depth-stencil-antialias.html
 context-lost-restored.html
 context-lost.html
 --max-version 1.9.9 context-type-test.html
+deleted-object-behavior.html
```

Comment:
Rafael, could you please add --min-version 1.0.4 here? This is just for bookkeeping, to know roughly when the new tests were added. Thanks.